.github/workflows/checks.yml

```diff
@@ -81,3 +81,42 @@ jobs:
         run: npm run typecheck
       - name: Run tests
         run: npm run test:unit
+
+  E2E:
+    # Only run e2e tests, if other jobs are successful
+    needs: [Build, Lint, Test]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: Install deps
+        run: npm install -g npm@7 && npm ci
+      - name: Start app in background
+        # Start app using Goerli testnet
+        run: VUE_APP_NETWORK=5 npm run serve &
+      - name: Checkout cypress app
+        uses: actions/checkout@v2
+        with:
+          repository: balancer-labs/frontend-e2e
```

Comment:
Should have a branch or tag param and an explanation of the workflow to add new tests or fix tests in the README. I imagine it'd be something like:

- Checkout branch of frontend-e2e + make changes
- Set that branch in .github/workflow/checks.yml 
- Merge frontend-e2e PR, then switch frontend PR to master and merge that.

Reply:
Updated the [readme](https://github.com/balancer-labs/frontend-e2e#adding-new-tests-and-fixing-tests)